CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_shopify_source v0.11.0
+[PR #78](https://github.com/fivetran/dbt_shopify_source/pull/78) introduces the following changes: 
+
+## 🚨 Breaking Changes 🚨
+- Added `source_relation` to the `partition_by` clauses that determine the `is_most_recent_record` in the `stg_shopify__metafield` and `stg_shopify__abandoned_checkout_discount_code` tables. If the user is leveraging the union feature, this could change data values.
```

Comment:
I'm seeing the col is called `index` in the abandoned_checkout table.
```suggestion
- Added `source_relation` to the `partition_by` clauses that determine the `is_most_recent_record` in the `stg_shopify__metafield` table and `index` in the `stg_shopify__abandoned_checkout_discount_code` table. If the user is leveraging the union feature, this could change data values.
```

Reply:
@fivetran-catfritz Updated!